inputs/costs/costs_co2.ad

```diff
@@ -1,11 +1,19 @@
-- query = UPDATE(AREA(), co2_price, USER_INPUT())
-- label = &euro;/tonne
-- label_query = Q(price_of_co2)
+
+# - label = &euro;/tonne
+# - label_query = Q(price_of_co2)
```

Comment:
If these lines are intended to be removed, please do so. The same for the blank lines following.

Reply:
I left them as comments as these lines refer to the RHS of the slider ( which should show the percentage change in fuel price) but this part of the issue is postponed for later as changes to the ETengine need to be made to facilitate this. 

So if these changes will be made at some point, I guess it's better to leave them as comments so that it's easier to fix. Do you agree?